2023-04-10 12:17:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DROP [*]DATABASE FARMFOLIO"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL OBJECTS, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
DROP DATABASE FARMFOLIO [42001-214]
2023-04-10 12:17:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE [*]DATABASE farmfolio"; expected "OR REPLACE, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL TEMPORARY, GLOBAL TEMPORARY, TEMP, TEMPORARY, TABLE, SYNONYM, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
CREATE DATABASE farmfolio [42001-214]
2023-04-10 12:18:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DROP [*]DATABASE FARMFOLIO"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL OBJECTS, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
DROP DATABASE FARMFOLIO [42001-214]
2023-04-10 12:18:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE [*]DATABASE farmfolio;"; expected "OR REPLACE, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL TEMPORARY, GLOBAL TEMPORARY, TEMP, TEMPORARY, TABLE, SYNONYM, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
CREATE DATABASE farmfolio; [42001-214]
2023-04-10 12:25:41 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 12:26:07 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at org.jdbi.v3.core.Jdbi.lambda$create$0(Jdbi.java:122)
	at org.jdbi.v3.core.Jdbi.open(Jdbi.java:319)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:365)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:83)
	at com.iloveyou.Main.main(Main.java:12)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2023-04-10 12:26:38 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:97)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:225)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 12:28:02 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at org.jdbi.v3.core.Jdbi.lambda$create$0(Jdbi.java:122)
	at org.jdbi.v3.core.Jdbi.open(Jdbi.java:319)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:365)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:83)
	at com.iloveyou.Main.main(Main.java:12)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2023-04-10 12:28:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DROP [*]DATABASE FARMFOLIO"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL OBJECTS, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
DROP DATABASE FARMFOLIO [42001-214]
2023-04-10 12:28:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE [*]DATABASE farmfolio;"; expected "OR REPLACE, FORCE, VIEW, ALIAS, SEQUENCE, USER, TRIGGER, ROLE, SCHEMA, CONSTANT, DOMAIN, TYPE, DATATYPE, AGGREGATE, LINKED, MEMORY, CACHED, LOCAL TEMPORARY, GLOBAL TEMPORARY, TEMP, TEMPORARY, TABLE, SYNONYM, UNIQUE, HASH, SPATIAL, INDEX"; SQL statement:
CREATE DATABASE farmfolio; [42001-214]
2023-04-10 12:29:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FARMFOLIO" not found; SQL statement:
DROP SCHEMA FARMFOLIO [90079-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:44)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:254)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1787)
	at org.jdbi.v3.core.result.ResultProducers.lambda$returningUpdateCount$0(ResultProducers.java:48)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:69)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:57)
	at org.jdbi.v3.core.Handle.execute(Handle.java:360)
	at org.jdbi.v3.core.Handle.execute(Handle.java:373)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:85)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:83)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:29:21 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:97)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:225)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 12:31:06 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at org.jdbi.v3.core.Jdbi.lambda$create$0(Jdbi.java:122)
	at org.jdbi.v3.core.Jdbi.open(Jdbi.java:319)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:365)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:83)
	at com.iloveyou.Main.main(Main.java:12)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2023-04-10 12:31:30 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:97)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:225)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 12:33:51 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at org.jdbi.v3.core.Jdbi.lambda$create$0(Jdbi.java:122)
	at org.jdbi.v3.core.Jdbi.open(Jdbi.java:319)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:365)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "FARMFOLIO" because "ENTITY, PRIMARY_KEY_7, CONSTRAINT_7" depends on it; SQL statement:
DROP SCHEMA FARMFOLIO [90107-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:62)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:254)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1787)
	at org.jdbi.v3.core.result.ResultProducers.lambda$returningUpdateCount$0(ResultProducers.java:48)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:69)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:57)
	at org.jdbi.v3.core.Handle.execute(Handle.java:360)
	at org.jdbi.v3.core.Handle.execute(Handle.java:373)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:86)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.CreateSchema.update(CreateSchema.java:48)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:190)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ENTITY" already exists; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY(ID INT NOT NULL,PRIMARY KEY (ID)) [42101-214]
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE(ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE(ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATUREENTITY_ID [*]INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATUREENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:34:09 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.CreateSchema.update(CreateSchema.java:48)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:190)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ENTITY" already exists; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY(ID INT NOT NULL,PRIMARY KEY (ID)) [42101-214]
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE(ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE(ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATUREENTITY_ID [*]INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATUREENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "FARMFOLIO" because "ENTITY, FEATURE_TYPE, SYSTEM_SEQUENCE_6D7C8FBB_F6CC_4EF1_B4D9_3E144D001E75, PRIMARY_KEY_D, PRIMARY_KEY_7, CONSTRAINT_D, CONSTRAINT_7" depends on it; SQL statement:
DROP SCHEMA FARMFOLIO [90107-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:62)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:254)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1787)
	at org.jdbi.v3.core.result.ResultProducers.lambda$returningUpdateCount$0(ResultProducers.java:48)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:69)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:57)
	at org.jdbi.v3.core.Handle.execute(Handle.java:360)
	at org.jdbi.v3.core.Handle.execute(Handle.java:373)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:86)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.CreateSchema.update(CreateSchema.java:48)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:190)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ENTITY" already exists; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY (ID INT NOT NULL,PRIMARY KEY (ID)) [42101-214]
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FEATURE_TYPE" already exists; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_TYPE (ID INT NOT NULL AUTO_INCREMENT,TYPE VARCHAR(45) NOT NULL,PRIMARY KEY (ID)) [42101-214]
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:34:39 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Schema "FARMFOLIO" already exists; SQL statement:
CREATE SCHEMA FARMFOLIO [90078-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.command.ddl.CreateSchema.update(CreateSchema.java:48)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:190)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:98)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "ENTITY" already exists; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY (ID INT NOT NULL,PRIMARY KEY (ID)) [42101-214]
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FEATURE_TYPE" already exists; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_TYPE (ID INT NOT NULL AUTO_INCREMENT,TYPE VARCHAR(45) NOT NULL,PRIMARY KEY (ID)) [42101-214]
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:35:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Cannot drop "FARMFOLIO" because "ENTITY, FEATURE_TYPE, SYSTEM_SEQUENCE_6D7C8FBB_F6CC_4EF1_B4D9_3E144D001E75, PRIMARY_KEY_D, PRIMARY_KEY_7, CONSTRAINT_D, CONSTRAINT_7" depends on it; SQL statement:
DROP SCHEMA FARMFOLIO [90107-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:632)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.command.ddl.DropSchema.update(DropSchema.java:62)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:254)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1787)
	at org.jdbi.v3.core.result.ResultProducers.lambda$returningUpdateCount$0(ResultProducers.java:48)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:69)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:57)
	at org.jdbi.v3.core.Handle.execute(Handle.java:360)
	at org.jdbi.v3.core.Handle.execute(Handle.java:373)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:85)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 12:36:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:36:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:36:02 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:95)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:95)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:37:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
2023-04-10 12:37:29 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:37:29 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:95)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.getSyntaxError(DbException.java:247)
	at org.h2.command.Parser.getSyntaxError(Parser.java:898)
	at org.h2.command.Parser.prepareCommand(Parser.java:572)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$0(Database.java:95)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:84)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:39:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
2023-04-10 12:39:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:39:42 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.getSyntaxError(DbException.java:247)
	at org.h2.command.Parser.getSyntaxError(Parser.java:898)
	at org.h2.command.Parser.prepareCommand(Parser.java:572)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:40:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
2023-04-10 12:40:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:40:19 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID))[*],UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID)),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.getSyntaxError(DbException.java:247)
	at org.h2.command.Parser.getSyntaxError(Parser.java:898)
	at org.h2.command.Parser.prepareCommand(Parser.java:572)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:40:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:40:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:40:58 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:41:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE);"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE); [42001-214]
2023-04-10 12:41:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:41:12 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE);"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE); [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE);"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT,FEATURE_KEY VARCHAR(45) NOT NULL,PRIMARY KEY (ID),UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE,UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE); [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:87)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:76)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:44:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:44:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:44:10 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:45:21 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL,FEATURE_ID VARCHAR(45) NOT NULL,FEATURE_TYPE_ID VARCHAR(45) NOT NULLDATA VARCHAR(45) NOT NULL,PRIMARY KEY (ID)FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID)FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID)FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:46:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:46:31 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:46:31 jdbc[3]: exception
org.h2.jdbc.JdbcBatchUpdateException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:800)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.getSyntaxError(DbException.java:261)
	at org.h2.command.Parser.getSyntaxError(Parser.java:900)
	at org.h2.command.Parser.read(Parser.java:5675)
	at org.h2.command.Parser.parseTableConstraintIf(Parser.java:9086)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:9321)
	at org.h2.command.Parser.parseCreateTable(Parser.java:9271)
	at org.h2.command.Parser.parseCreate(Parser.java:6784)
	at org.h2.command.Parser.parsePrepared(Parser.java:763)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:666)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:186)
	at org.h2.jdbc.JdbcStatement.executeBatchElement(JdbcStatement.java:842)
	at org.h2.jdbc.JdbcStatement.executeBatch(JdbcStatement.java:794)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.Batch.execute(Batch.java:86)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:88)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:77)
	at com.iloveyou.Main.main(Main.java:12)
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE [*]ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42000-214]
2023-04-10 12:47:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(45) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:50:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:51:28 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY) ASC VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL) ASC VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY) ASC VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL) ASC VISIBLE) [42001-214]
2023-04-10 12:51:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL) VISIBLE) [42001-214]
2023-04-10 12:52:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL,PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 12:54:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 13:12:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), UNIQUE [*]INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE)"; expected "("; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), UNIQUE INDEX FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) VISIBLE, UNIQUE INDEX FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) VISIBLE) [42001-214]
2023-04-10 13:16:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), FEATURE_KEY_UNIQUE [*](FEATURE_KEY ASC) UNIQUE INDEX VISIBLE, FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE)"; expected "IDENTITY, BIGSERIAL, SERIAL, data type"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID), FEATURE_KEY_UNIQUE (FEATURE_KEY ASC) UNIQUE INDEX VISIBLE, FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE) [42001-214]
2023-04-10 13:21:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE UNIQUE [*]VISIBLE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC);FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE)"; expected "HASH, INDEX"; SQL statement:
CREATE UNIQUE VISIBLE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC);FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE) [42001-214]
2023-04-10 13:22:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC) [*]UNIQUE VISIBLE;FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE)"; SQL statement:
CREATE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC) UNIQUE VISIBLE;FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE) [42000-214]
2023-04-10 13:22:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC) [*]UNIQUE VISIBLE;FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE)"; SQL statement:
CREATE INDEX FEATURE_KEY_UNIQUE ON FARMFOLIO.FEATUER (FEATURE_KEY ASC) UNIQUE VISIBLE;FEATURE_LABEL_UNIQUE (FEATURE_LABEL ASC) UNIQUE INDEX VISIBLE) [42000-214]
2023-04-10 13:24:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "FARMFOLIO_FEATURE" not found; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE (ID INT NOT NULL AUTO_INCREMENT, FEATURE_KEY VARCHAR(64) NOT NULL, FEATURE_LABEL VARCHAR(64) NOT NULL, PRIMARY KEY (ID)) [42102-214]
2023-04-10 13:25:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE (ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL [*]DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; expected "AS, DEFAULT, GENERATED, ON UPDATE, AUTO_INCREMENT, DEFAULT ON NULL, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, COMMENT, PRIMARY KEY, UNIQUE, NOT NULL, NULL, CHECK, REFERENCES, AUTO_INCREMENT, ,, )"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE (ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42001-214]
2023-04-10 13:25:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.ENTITY_FEATURE (ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL, DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) [*]FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE (ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL, DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID) FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID) FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID) FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42001-214]
2023-04-10 13:26:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "ID" not found; SQL statement:
CREATE TABLE FARMFOLIO.ENTITY_FEATURE (ENTITY_ID INT NOT NULL, FEATURE_ID VARCHAR(45) NOT NULL, FEATURE_TYPE_ID VARCHAR(45) NOT NULL, DATA VARCHAR(45) NOT NULL, PRIMARY KEY (ID), FOREIGN KEY (ENTITY_ID) REFERENCES ENTITY(ID), FOREIGN KEY (FEATURE_ID) REFERENCES FEATURE(ID), FOREIGN KEY (FEATURE_TYPE_ID) REFERENCES FEATURE_TYPE(ID)) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:749)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:184)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:119)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.ddl.AlterTable.update(AlterTable.java:46)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:134)
	at org.h2.command.CommandContainer.update(CommandContainer.java:169)
	at org.h2.command.Command.executeUpdate(Command.java:252)
	at org.h2.jdbc.JdbcPreparedStatement.execute(JdbcPreparedStatement.java:254)
	at org.jdbi.v3.core.statement.SqlLoggerUtil.wrap(SqlLoggerUtil.java:31)
	at org.jdbi.v3.core.statement.SqlStatement.internalExecute(SqlStatement.java:1787)
	at org.jdbi.v3.core.result.ResultProducers.lambda$returningUpdateCount$0(ResultProducers.java:48)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:69)
	at org.jdbi.v3.core.statement.Update.execute(Update.java:57)
	at org.jdbi.v3.core.Handle.execute(Handle.java:360)
	at org.jdbi.v3.core.Handle.execute(Handle.java:373)
	at com.iloveyou.domain.persistence.Database.lambda$1(Database.java:84)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:371)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:78)
	at com.iloveyou.Main.main(Main.java:12)
2023-04-10 13:27:07 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:97)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:225)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 13:27:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "KEY_SEQ" not found [42122-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:337)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetImpl.getInt(JDBCResultSetImpl.java:500)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils.safeGetInt(JDBCUtils.java:110)
	at org.jkiss.dbeaver.ext.generic.model.GenericUtils.safeGetInt(GenericUtils.java:84)
	at org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel.createConstraintColumnsImpl(GenericMetaModel.java:819)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createConstraintColumnsImpl(H2MetaModel.java:178)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:98)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:360)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getConstraints(GenericTableBase.java:222)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:146)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:351)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getAssociations(GenericTableBase.java:253)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.extractPropertyValue(DBNDatabaseNode.java:964)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1036)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:189)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadTreeItems(DBNDatabaseNode.java:595)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadChildren(DBNDatabaseNode.java:464)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 13:27:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "KEY_SEQ" not found [42122-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:337)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetImpl.getInt(JDBCResultSetImpl.java:500)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils.safeGetInt(JDBCUtils.java:110)
	at org.jkiss.dbeaver.ext.generic.model.GenericUtils.safeGetInt(GenericUtils.java:84)
	at org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel.createConstraintColumnsImpl(GenericMetaModel.java:819)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createConstraintColumnsImpl(H2MetaModel.java:178)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:98)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:360)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getConstraints(GenericTableBase.java:222)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:146)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:351)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getAssociations(GenericTableBase.java:253)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.extractPropertyValue(DBNDatabaseNode.java:964)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1036)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:189)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadTreeItems(DBNDatabaseNode.java:595)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadChildren(DBNDatabaseNode.java:464)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 13:27:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "KEY_SEQ" not found [42122-210]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:521)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:496)
	at org.h2.message.DbException.get(DbException.java:227)
	at org.h2.message.DbException.get(DbException.java:203)
	at org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3492)
	at org.h2.jdbc.JdbcResultSet.getInt(JdbcResultSet.java:337)
	at org.jkiss.dbeaver.model.impl.jdbc.exec.JDBCResultSetImpl.getInt(JDBCResultSetImpl.java:500)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCUtils.safeGetInt(JDBCUtils.java:110)
	at org.jkiss.dbeaver.ext.generic.model.GenericUtils.safeGetInt(GenericUtils.java:84)
	at org.jkiss.dbeaver.ext.generic.model.meta.GenericMetaModel.createConstraintColumnsImpl(GenericMetaModel.java:819)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createConstraintColumnsImpl(H2MetaModel.java:178)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:98)
	at org.jkiss.dbeaver.ext.generic.model.ConstraintKeysCache.fetchObjectRow(ConstraintKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:360)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getConstraints(GenericTableBase.java:222)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:146)
	at org.jkiss.dbeaver.ext.generic.model.ForeignKeysCache.fetchObject(ForeignKeysCache.java:1)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.loadObjects(JDBCCompositeCache.java:351)
	at org.jkiss.dbeaver.model.impl.jdbc.cache.JDBCCompositeCache.getObjects(JDBCCompositeCache.java:120)
	at org.jkiss.dbeaver.ext.generic.model.GenericTableBase.getAssociations(GenericTableBase.java:253)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.extractPropertyValue(DBNDatabaseNode.java:964)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1036)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode$PropertyValueReader.run(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.exec.DBExecUtils.tryExecuteRecover(DBExecUtils.java:189)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadTreeItems(DBNDatabaseNode.java:595)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.loadChildren(DBNDatabaseNode.java:464)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:227)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-04-10 13:28:11 database: flush
org.h2.message.DbException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:395)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:125)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:3318)
	at org.h2.mvstore.MVStore.panic(MVStore.java:593)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:469)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at org.jdbi.v3.core.Jdbi.lambda$create$0(Jdbi.java:122)
	at org.jdbi.v3.core.Jdbi.open(Jdbi.java:319)
	at org.jdbi.v3.core.Jdbi.withHandle(Jdbi.java:365)
	at com.iloveyou.domain.persistence.Database.resetDatabase(Database.java:79)
	at com.iloveyou.Main.main(Main.java:12)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: General error: "org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]" [50000-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 22 more
Caused by: org.h2.mvstore.MVStoreException: The file is locked: C:/Users/Ethan/data/school/190/Farmfolio/api/resources/db.mv.db [2.1.214/7]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:178)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	... 16 more
2023-04-10 13:30:17 jdbc[3]: exception
java.sql.SQLClientInfoException: Client info name 'ApplicationName' not supported.
	at org.h2.jdbc.JdbcConnection.setClientInfo(JdbcConnection.java:1573)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.populateClientAppName(GenericDataSource.java:183)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.openConnection(GenericDataSource.java:146)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.openConnection(H2DataSource.java:74)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCExecutionContext.connect(JDBCExecutionContext.java:103)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.initializeMainContext(JDBCRemoteInstance.java:102)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCRemoteInstance.<init>(JDBCRemoteInstance.java:61)
	at org.jkiss.dbeaver.model.impl.jdbc.JDBCDataSource.initializeRemoteInstance(JDBCDataSource.java:130)
	at org.jkiss.dbeaver.ext.generic.model.GenericDataSource.<init>(GenericDataSource.java:109)
	at org.jkiss.dbeaver.ext.h2.model.H2DataSource.<init>(H2DataSource.java:43)
	at org.jkiss.dbeaver.ext.h2.model.H2MetaModel.createDataSourceImpl(H2MetaModel.java:55)
	at org.jkiss.dbeaver.ext.generic.GenericDataSourceProvider.openDataSource(GenericDataSourceProvider.java:106)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.openDataSource(DataSourceDescriptor.java:1191)
	at org.jkiss.dbeaver.registry.DataSourceDescriptor.connect(DataSourceDescriptor.java:1056)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.run(ConnectJob.java:70)
	at org.jkiss.dbeaver.runtime.jobs.ConnectJob.runSync(ConnectJob.java:97)
	at org.jkiss.dbeaver.ui.actions.datasource.DataSourceHandler.connectToDataSource(DataSourceHandler.java:109)
	at org.jkiss.dbeaver.ui.actions.datasource.UIServiceConnectionsImpl.initConnection(UIServiceConnectionsImpl.java:87)
	at org.jkiss.dbeaver.model.navigator.DBNDataSource.initializeNode(DBNDataSource.java:158)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:225)
	at org.jkiss.dbeaver.model.navigator.DBNDatabaseNode.getChildren(DBNDatabaseNode.java:1)
	at org.jkiss.dbeaver.model.navigator.DBNUtils.getNodeChildrenFiltered(DBNUtils.java:80)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:49)
	at org.jkiss.dbeaver.ui.navigator.database.load.TreeLoadService.evaluate(TreeLoadService.java:1)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:88)
	at org.jkiss.dbeaver.ui.LoadingJob.run(LoadingJob.java:72)
	at org.jkiss.dbeaver.model.runtime.AbstractJob.run(AbstractJob.java:105)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
2023-09-04 10:47:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, [*]KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC);"; expected "identifier"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC); [42001-214]
2023-09-04 10:50:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, [*]KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC);"; expected "identifier"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC); [42001-214]
2023-09-04 10:54:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, [*]KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC);"; expected "identifier"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC); [42001-214]
2023-09-04 10:54:46 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, [*]KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC);"; expected "identifier"; SQL statement:
CREATE TABLE FARMFOLIO.FEATURE_KEY (ID INT NOT NULL AUTO_INCREMENT, KEY VARCHAR(64) NOT NULL, LABEL VARCHAR(64) NOT NULL, FEATURE_TYPE_ID INT NOT NULLPRIMARY KEY (ID))CREATE UNIQUE INDEX KEY_UNIQUE ON FARMFOLIO.FEATURE_KEY (KEY ASC);CREATE UNIQUE INDEX LABEL_UNIQUE ON FARMFOLIO.FEATURE_KEY (LABEL ASC); [42001-214]
